torch/csrc/generic/Tensor.cpp

```diff
@@ -816,6 +816,8 @@ PyTypeObject THPTensorStatelessType = {
 void THPTensor_(initCopyMethods)()
 {
   auto& h = THTensor_(copy_functions);
+  // copy from same type
+  THPInsertCopyFunction(h, &THTensor_(copy));
```

Comment:
Eh I think we should clean this up somehow 😕 I don't know why isn't any of the lines below overwriting this one.